Tensile/KernelWriterAssembly.py

```diff
@@ -11997,8 +11999,7 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
       vc0 = element[3]
 
       kStr += addrCalc.emitAddressSetupCode(kernel, ss, 'C', tmpVgpr, tmpS01, edge, beta, atomic, elementIdx, addrDVgpr)
-
-      if edge:
+      if edge and (kernel["BufferStore"] or beta):
```

Comment:
What about edge + BufferStore + not beta case?
edgeProtectCode will not be called, right?